packages/app/src/Spotlights.tsx

```diff
@@ -0,0 +1,168 @@
+    logViews.forEach(logView => {
+      logViewActions.push({
+        group: 'Saved searches',
+        icon: <i className="bi bi-layout-text-sidebar-reverse" />,
+        description: logView.query,
+        title: logView.name,
+        keywords: ['search', 'log', 'saved'],
+        onTrigger: () => {
+          router.push(`/search/${logView._id}`);
+        },
+      });
+    });
+
+    // Dashboards
+    dashboards.forEach(dashboard => {
+      logViewActions.push({
+        group: 'Dashboards',
+        icon: <i className="bi bi-grid-1x2" />,
+        title: dashboard.name,
+        keywords: ['dashboard'],
+        onTrigger: () => {
+          router.push(`/dashboards/${dashboard._id}`);
+        },
+      });
+    });
+
+    logViewActions.push(
+      {
+        group: 'Menu',
+        icon: <i className="bi bi-layout-text-sidebar-reverse" />,
+        title: 'Search',
+        description: 'Start a new search',
+        keywords: ['log', 'events', 'logs'],
+        onTrigger: () => {
+          router.push('/search');
+        },
+      },
+      {
+        group: 'Menu',
+        icon: <i className="bi bi-graph-up" />,
+        title: 'Chart Explorer',
+        description: 'Explore your data',
+        keywords: ['graph', 'metrics'],
+        onTrigger: () => {
+          router.push('/chart');
+        },
+      },
+      {
+        group: 'Menu',
+        icon: <i className="bi bi-grid-1x2" />,
+        title: 'New Dashboard',
+        description: 'Create a new dashboard',
+        keywords: ['graph'],
+        onTrigger: () => {
+          router.push('/dashboards');
+        },
+      },
+      {
+        group: 'Menu',
+        icon: <i className="bi bi-laptop" />,
+        title: 'Client Sessions',
+        description: 'View client sessions',
+        keywords: ['browser', 'web'],
+        onTrigger: () => {
+          router.push('/sessions');
+        },
+      },
+      {
+        group: 'Menu',
+        icon: <i className="bi bi-bell" />,
+        title: 'Alerts',
+        description: 'View and manage alerts',
+        onTrigger: () => {
+          router.push('/alerts');
+        },
+      },
+      ...(SERVICE_DASHBOARD_ENABLED
+        ? [
+            {
+              group: 'Menu',
+              title: 'Service Health',
+              icon: <i className="bi bi-heart-pulse" />,
+              description: 'HTTP, Database and Infrastructure metrics',
+              onTrigger: () => {
+                router.push('/services');
+              },
+            },
+          ]
+        : []),
+      {
+        group: 'Menu',
+        icon: <i className="bi bi-gear" />,
+        title: 'Team Settings',
+
+        onTrigger: () => {
+          router.push('/team');
+        },
+      },
+      {
+        group: 'Menu',
+        icon: <i className="bi bi-question-circle" />,
+        title: 'Documentation',
+        keywords: ['help', 'docs'],
+        onTrigger: () => {
+          router.push('/help');
```

Comment:
this should go to https://www.hyperdx.io/docs

Reply:
fixed